{"name": "Создание пользователя который уже есть в системе", "status": "passed", "description": "При создании дублирующего пользователя срабатывает allert ", "start": 1744200573693, "stop": 1744200574150, "uuid": "16976dfa-23a3-481d-8827-a749309c0a17", "historyId": "511af5c3d1ba0594e424965a597653ae", "testCaseId": "511af5c3d1ba0594e424965a597653ae", "fullName": "tests.test_create_user.TestCreateUser#test_create_double_user_error", "labels": [{"name": "suite", "value": "Создание пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "DESKTOP-GVJUSQ6"}, {"name": "thread", "value": "6996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}